src/compas_fab/robots/tool.py

```diff
@@ -64,6 +65,7 @@ def link_name(self):
     @link_name.setter
     def link_name(self, link_name):
         self.tool_model.link_name = link_name
+        self.attached_collision_meshes = self.get_attached_collision_meshes()  # rebuild ACMs
```

Comment:
maybe this should just be a property that is built whenever it is accessed...

Reply:
moved it to a property on https://github.com/compas-dev/compas_fab/pull/240/commits/b47a15741ac172fb4cfad9a483de133917d2625e